src/net_io.c

```diff
@@ -376,7 +376,8 @@ static gboolean net_io_do_async(net_io_request_t *request) {
 
   /* create worker thread */
   request->refcount = 2;   // master and worker hold a reference
-  if(!g_thread_create(&worker_thread, request, FALSE, NULL) != 0) {
+  /* if(!g_thread_create(&worker_thread, request, FALSE, NULL) != 0) { */
+  if(g_thread_try_new("worker", &worker_thread, request, NULL) == NULL) {
```

Comment:
The function `g-thread-create` is deprecated and should be replaced by  `g_thread_new` or  `g_thread_try_new`. Using this deprecated function causes warnings:

```
../maep-qt/src/net_io.c: In function 'net_io_do_async':
../maep-qt/src/net_io.c:379:3: warning: 'g_thread_create' is deprecated (declared at /usr/include/glib-2.0/glib/deprecated/gthread.h:100): Use 'g_thread_new' instead [-Wdeprecated-declarations]
   if(!g_thread_create(&worker_thread, request, FALSE, NULL) != 0) {
```
See also https://developer.gnome.org/glib/stable/glib-Deprecated-Thread-APIs.html#g-thread-create